<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shortcuts</class>
 <widget class="QDialog" name="shortcuts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shortcuts</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="3">
    <widget class="QTableWidget" name="tableWidget_shortcuts">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>80</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>13</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>13</number>
     </attribute>
     <column>
      <property name="text">
       <string>Key</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Action</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton_add">
     <property name="text">
      <string>Add...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_delete">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>shortcuts</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>shortcuts</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
